src/Riok.Mapperly/Helpers/EnumMappingBuilderHelper.cs

```diff
@@ -0,0 +1,86 @@
+using Microsoft.CodeAnalysis;
+using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Descriptors;
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Helpers;
+
+public static class EnumMappingBuilderHelper
+{
+    public static HashSet<IFieldSymbol> GetMembers(this ITypeSymbol symbol, HashSet<IFieldSymbol> ignoredMembers) =>
+        symbol.GetMembers().OfType<IFieldSymbol>().Where(x => !ignoredMembers.Remove(x)).ToHashSet(SymbolTypeEqualityComparer.FieldDefault);
```

Comment:
This should probably be moved to the symbol accessor to make use of the caching.

Reply:
Yes, agreed. I have moved this to the SymbolAccessor.